datahub-web-react/src/app/shared/tags/AddTagsTermsModal.tsx

```diff
@@ -292,12 +288,76 @@ export default function AddTagsTermsModal({
                 message.error({ content: `Failed to add: \n ${e.message || ''}`, duration: 3 });
             })
             .finally(() => {
-                setDisableAdd(false);
+                setDisableAction(false);
                 onCloseModal();
                 setUrns([]);
             });
     };
 
+    const batchRemoveTags = () => {
+        batchRemoveTagsMutation({
+            variables: {
+                input: {
+                    tagUrns: urns,
+                    resources,
+                },
+            },
+        })
+            .then(({ errors }) => {
+                if (!errors) {
+                    message.success({
+                        content: `Removed ${type === EntityType.GlossaryTerm ? 'Terms' : 'Tags'}!`,
+                        duration: 2,
+                    });
+                }
+            })
+            .catch((e) => {
+                message.destroy();
+                message.error({ content: `Failed to remove: \n ${e.message || ''}`, duration: 3 });
+            })
+            .finally(() => {
+                setDisableAction(false);
+                onCloseModal();
+                setUrns([]);
+            });
+    };
+
+    const batchRemoveTerms = () => {
+        // Not yet supported
+    };
+
+    const editTags = () => {
+        if (operationType === OperationType.ADD) {
+            batchAddTags();
+        } else {
+            batchRemoveTags();
+        }
+    };
+
+    const editTerms = () => {
+        if (operationType === OperationType.ADD) {
+            batchAddTerms();
+        } else {
+            batchRemoveTerms();
+        }
+    };
+
+    // Function to handle the modal action's
+    const onOk = () => {
+        console.log(entityType); // TODO: Add analytics
```

Comment:
nit: console.log

Reply:
good call.. i need to remove lint n this variable